docs/deploy.md

```diff
@@ -135,3 +135,51 @@ frontend:
 | /<*>.md        | /<*>.md        | 200 (Rewrite) |
 | /<*>.png       | /<*>.png       | 200 (Rewrite) |
 | /<*>           | /index.html    | 200 (Rewrite) |        
+
+
+## Docker
+
+>There is a simple demo for building docsify in docker.
+
+?> tips: you can create all the files for this demo from [Quickstart](https://docsify.js.org/#/quickstart?id=initialize).
+
+- Preparing Docsify initial files  
+  You need prepare the initial files (generated by run `docsify init ./docs`) instead of making in container.
```

Comment:
To prevent outdated instructions here, let's just refer people to the Quickstart guide: "See the Quickstart section for instructions on how to create these files manually or using docsify-cli."

(Quickstart section and docsify-cli should be linked)